src/js/actions/actionCreator.js

```diff
@@ -68,3 +54,36 @@ export const loadMovies = () => (dispatch, getState) => {
     .then( response => dispatch(loadMoviesSuccess(response)))
     .catch( error => dispatch(loadMoviesFailed(error)));
 };
+
+export const loadMoviesFailed = () => ({
+  type: actionTypes.LOAD_MOVIES_FAILED,
+});
+
+export const loadMovieSuccess = (response) => {
```

Comment:
replace `response` with `movie` and delete line 63

Reply:
yes! 👍 thats remained from copypaste